src/handlers/xp.js

```diff
@@ -13,10 +21,10 @@ const filterAccount = (obj, targetUserDiscordID) => {
       const parts = NodeAddress.toParts(alias.address)
       return parts.indexOf('discord') > 0
     })
-  if (discordAlias.length === 1) {
+  if (discordAlias.length >= 1) {
     // Retrieve the Discord ID
-    const discordId = NodeAddress.toParts(discordAlias[0].address)[4]
-    if (discordId === targetUserDiscordID) {
+    const discordId = discordAlias.find(discordAccount => filterMultipleDiscordAccounts(discordAccount, targetUserDiscordID))
+    if (discordId !== undefined){
```

Comment:
See NIT above

Reply:
`Array.prototype.find` returns undefined if no element was found (https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/find), therefore I think it's relevant to return this instead of null.